state.go

```diff
@@ -181,7 +188,7 @@ func (state *KatzenmintState) Commit() ([]byte, error) {
 	state.Lock()
 	defer state.Unlock()
 
-	var errDoc error
+	var errDoc error = nil
```

Comment:
Might better to rename errDoc, also I think default errDoc should be nil.

Reply:
Fixed